libs/blocks/section-metadata/section-metadata.js

```diff
@@ -7,7 +7,12 @@ function handleBackground(div, section) {
   } else {
     const color = div.textContent;
     if (color) {
-      section.style.backgroundColor = color;
+      if (color.indexOf('gradient') !== -1) {
+        section.style.background = color;
+      }
+      else {
+        section.style.backgroundColor = color;
+      }
```

Comment:
Since setting `background: colorName;` is the same as `backgroundColor: colorName` we should probably just place the `background` string directly.  This allows for any kind of background strings.

```
const background = div.textContent;
if (background) {
  section.style.background = background;
}
```

@rgclayton @auniverseaway @ryanmparrish Thoughts?

Reply:
@chrischrischris Thanks for sharing the feedback. I have updated the PR to set background property in place of backgroundColor to allow all background strings. Could you please review?

Reply:
@chrischrischris Thanks for adding the approval for the PR and reviewing the latest changes. 
Marking the conversation as resolved.